src/ApiCore/TransportFactory.php

```diff
@@ -136,35 +77,36 @@ public static function build(array $args)
                         'gRPC extension please see https://cloud.google.com/php/grpc.'
                     );
                 }
-                $stubOpts = [
-                    'force_new' => $args['forceNewChannel']
-                ];
-                // We need to use array_key_exists here because null is a valid value
-                if (!array_key_exists('sslCreds', $args)) {
-                    $stubOpts['credentials'] = self::createSslChannelCredentials();
-                } else {
-                    $stubOpts['credentials'] = $args['sslCreds'];
-                }
+
+                $stubOpts = ['credentials'=> self::createSslChannelCredentials()];
 
                 return new GrpcTransport(
-                    $host,
+                    $serviceAddress,
                     $authWrapper,
-                    $stubOpts,
-                    $args['channel']
```

Comment:
Why are we removing the capability to provide a channel, override the `force_new` option, and to provide ssl credentials?